HTML/text.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="../CSS/reset.css">
+    <link rel="stylesheet" href="../CSS/stylesheet.css">
+    <link rel="stylesheet" href="../CSS/responsive.css">
+    <link href="https://fonts.googleapis.com/css?family=Noto+Sans+JP&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <img src="../img_pc/QuelBREAD.png" alt="logo">
+        <nav>
+            <li><a href="../HTML/text.html">TEXT</a></li>
+            <li><a href="../HTML/position.html">POSITION</a></li>
+            <li><a href="../HTML/flexbox.html">FLEXBOX</a></li>
+            <li><a href="../HTML/hover.html">HOVER</a></li>
+            <li><a href="../HTML/accordion.html">ACCORDION</a></li>
+            <li><a href="../HTML/slideshow.html">SLIDE SHOW</a></li>
+        </nav>
+    </header>
+    <main>
+        <section class="article_why">
+            <h1>なぜハードパン？</h1>
+            <p>「 <span>愛</span>、それだけです。」</p>
+            <p>普段ハードパンを食べないような人にもハードパンの面白さと美味しさを知って欲しいという想い<br>
+                からフランスに行き、８年間ひたすらハードパンをつくりつづけ、深まり溢れる愛情を止めること<br>
+                できず今回東京で出店する運びとなりました。
+            </p>
```

Comment:
```suggestion
            <p>
            　普段ハードパンを食べないような人にもハードパンの面白さと美味しさを知って欲しいという想い<br>
                からフランスに行き、８年間ひたすらハードパンをつくりつづけ、深まり溢れる愛情を止めること<br>
                できず今回東京で出店する運びとなりました。
            </p>
```
複数行はこうしたほうが見やすいです（1行なら改行させない）

Reply:
あと`br`いらないですね

Reply:
ご指摘ありがとうございます！
bf5b70782da35afe826e952d1fc4932edc02772d
このコミットで修正しました！